Makefile

```diff
@@ -1,7 +1,14 @@
 NAMESPACE=app-metrics
+APP_NAMESPACES=app-metrics-apps
```

Comment:
this is different to the default in the operator.yaml, not sure if that's important.

Reply:
You are correct. :) It's done this way so that the operator can be decoupled from MDC in the community if someone wishes to do that. I think I need to update the value passed to the operator env var, though 🤔